Scripts/Runtime/Logging/.CSProject/UnityLogListener.cs

```diff
@@ -307,10 +306,15 @@ private void SendToLogger(UnityEngine.Object context, LogLevel logLevel, string
         {
             (StackFrame callerFrame, MethodBase callerMethod) = ResolveCaller(exception);
 
-            string callerPath = callerFrame?.GetFileName() ?? callerMethod?.ReflectedType.Name;
+            string callerMethodName = callerMethod?.Name;
+            string callerFilePath = callerFrame?.GetFileName();
+            int callerLineNumber = callerFrame?.GetFileLineNumber() ?? 0;
 
-            Logger logger = context != null ? Log.GetLogger(context) : Log.GetStaticLogger(ResolveContextFromMethod(callerMethod));
-            logger.AtLevel(logLevel, message, callerPath, callerMethod?.Name, callerFrame?.GetFileLineNumber() ?? 0);
+            Logger logger = context != null
+                ? Log.GetLogger(context)
+                : Log.GetStaticLogger(callerMethod?.ReflectedType ?? typeof(UnknownContext));
+
+            logger.AtLevel(logLevel, message, callerMethodName, callerFilePath, callerLineNumber);
```

Comment:
This change came from the rework in https://github.com/decline-cookies/anvil-csharp-core/pull/123, namely the inclusion of the logger's `ownerTypeName` in the log context.

Previously, if no file path was available (i.e. for DLL code) it would fall back on the method's owner type. This no longer makes sense as the caller type is now also included in the log context, and furthermore Unity's `context` object passed to `LogException` would not actually be the source of the exception, but the last non-DLL function in the stack. For example, creating a second `UnityLogListener` instance from a "Demo.cs" script would result in:
```
(Demo..ctor|UnityLogListener) System.Exception: There can only be one instance [...]
```
...but with these changes, the missing file path is left blank, and the context falls back on the calling method's owner type, resulting in:
```
(UnityLogListener..ctor|<unknown>) System.Exception: There can only be one instance [...]
```

Reply:
👍